v3/plugins/redhat/vscode-camelk/0.0.10/meta.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-camelk
+version: 0.0.10
+type: VS Code extension
+displayName: Tooling for Apache Camel K
+title: Tooling for Apache Camel K
+description: VS Code support for Apache Camel K functionality
```

Comment:
Since this is a Che plugin here, it may be confusing to say it provides "VS Code support".